src/Illuminate/Support/Arr.php

```diff
@@ -120,7 +120,7 @@ public static function exists($array, $key)
             return $array->offsetExists($key);
         }
 
-        return array_key_exists($key, $array);
+        return isset($array[$key]) || array_key_exists($key, $array);
```

Comment:
array_key_exists is pretty much the same speed on php 7, so you've actually made this slower